README.rst

```diff
@@ -13,43 +14,23 @@ Websites
 * Source code: http://github.com/h5py/h5py
 * Mailing list: https://groups.google.com/d/forum/h5py
 
-For advanced installation options, see http://docs.h5py.org.
+Installation
+------------
 
-Prerequisites
--------------
+Pre-build `h5py` can either be installed via your Python Distribution (e.g.
+`Continuum Anaconda`_, `Enthought Canopy`_) or from `PyPI`_ via `pip`_.
 
-You need, at a minimum:
+More detailed installation instructions, including how to install `h5py` with
+MPI support, can be found at: http://docs.h5py.org/en/latest/build.html.
```

Comment:
I would use internal links here 

```
:ref:`build`
```
here

and 

```
.. _build:
```

over the section you want to link ot.